match/plaquemines_so.py

```diff
@@ -33,11 +34,34 @@ def match_post(cprr, post):
     return cprr
 
 
+def match_pprr_and_post(pprr, post):
+    dfa = pprr[['uid', 'first_name', 'last_name']]
+    dfa.loc[:, 'fc'] = dfa.first_name.fillna('').map(lambda x: x[:1])
+    dfa = dfa.drop_duplicates(subset=['uid']).set_index('uid')
+
+    dfb = post[['uid', 'first_name', 'last_name']]
+    dfb.loc[:, 'fc'] = dfb.first_name.fillna('').map(lambda x: x[:1])
+    dfb = dfb.drop_duplicates(subset=['uid']).set_index('uid')
+
+    matcher = ThresholdMatcher(ColumnsIndex(["fc"]), {
+        "last_name": JaroWinklerSimilarity(),
+        "first_name": JaroWinklerSimilarity(),
+    }, dfa, dfb)
+    decision = 0.93
+    matcher.save_pairs_to_excel(data_file_path(
+        "match/plaquemines_so_pprr_2018_v_post_pprr_2020_11_06.xlsx"), decision)
+    matches = matcher.get_index_pairs_within_thresholds(lower_bound=decision)
+    return extract_events_from_post(post, matches, 'Plaquemines SO')
+
+
 if __name__ == '__main__':
     cprr = pd.read_csv(data_file_path('clean/cprr_plaquemines_so_2019.csv'))
+    pprr = pd.read_csv(data_file_path('clean/pprr_plaquemines_so_2018.csv'))
     post = prepare_post_data()
-    cprr = match_post(cprr, post)
+    cprr = match_cprr_post(cprr, post)
```

Comment:
Perhaps we should match CPRR against PPRR instead since the roster now comes from PPRR.

Reply:
Yes. Thank you.